app/controllers/submissions_controller.rb

```diff
@@ -159,6 +167,14 @@ def destroy
     end
   end
 
+  # POST
+  def soft_delete
+    @submission.soft_deleted = true
+    @submission.save
```

Comment:
You should use `.update(soft_deleted: true)`. Also, you should use `save!`/`update!` because we don't want whoever did it to think that it worked when it really didn't.